Dockerfile

```diff
@@ -1,15 +1,15 @@
-FROM azul/zulu-openjdk-alpine:17.0.6-17.40.19-jre as builder
+FROM azul/zulu-openjdk-alpine:17.0.4.1-17.36.17-jre as builder
```

Comment:
why the downgrade? 
The newest tag is `17.0.8.1-17.44.53-jre`

Reply:
Was a mistake. The readme of the base image on Dockerhub was wrong and I did not double-check.